dig_core/lib/src/domain/usecase/check_mnemonic_valid_usecase.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:cosmos_utils/mnemonic.dart';
+import 'package:dig_core/dig_core.dart';
+import 'package:dartz/dartz.dart';
+import 'package:dig_core/src/domain/usecase/usecase.dart';
+import 'package:bip39/bip39.dart' as bip39;
+import 'package:injectable/injectable.dart';
+import 'package:logger/logger.dart';
+
+@injectable
+class CheckMnemonicValidUseCase
+    extends SyncUseCase<bool, CheckMnemonicValidParam> {
+  @override
+  Either<BaseDigException, bool> call(CheckMnemonicValidParam params) {
+    try {
+      if (params.mnemonic.isEmpty) {
+        return const Left(DigException(message: 'Mnemonic cannot be empty'));
+      } else if (!RegExp(r'^[a-zA-Z ]+$').hasMatch(params.mnemonic)) {
+        return const Left(DigException(message: 'Invalid character'));
+      } else if (![12, 24].contains(params.mnemonic.splitToWords().length)) {
```

Comment:
12 and 24 is just common cases. there are more like 32, ... so we rather warn user that this mnemonic is not created by dig app than preventing them to import. So I think u should remove this case

Reply:
I mean just remove this case for now. The warning part for the future

Reply:
I agree